### PR TITLE
Docs: update schema change capability

### PR DESCRIPTION
**Pull request by nataliekwong**
## What
Updates docs to reflect latest capabilities of the schema change and management feature.

### CODE REVIEW

File: docs/cloud/managing-airbyte-cloud/manage-schema-changes.md [hunk 1]
```diff
@@ -1,12 +1,27 @@
 # Manage schema changes
 
-Once every 24 hours, Airbyte checks for changes in your source schema and allows you to review the changes and fix breaking changes. This process helps ensure accurate and efficient data syncs, minimizing errors and saving you time and effort in managing your data pipelines.
+You can specify for each connection how Airbyte should handle any change of schema in the source. This process helps ensure accurate and efficient data syncs, minimizing errors and saving you time and effort in managing your data pipelines.
 
-:::note 
+Airbyte checks for any changes in your source schema before every sync or once every 24 hours, whichever is more frequent.
```
Comment (nataliekwong) [inline docs/cloud/managing-airbyte-cloud/manage-schema-changes.md:5]: @mfsiega-airbyte I think I recall this being the case, but am not sure now if this is true. Is it still once every 24 hours or does it check on sync start as well?

Comment (nataliekwong) [reply docs/cloud/managing-airbyte-cloud/manage-schema-changes.md]: Updated to say every 24 hours

Comment (mfsiega-airbyte) [reply docs/cloud/managing-airbyte-cloud/manage-schema-changes.md]: I would phrase it as: `Airbyte checks for any changes in your source schema before syncing, at most once every 24 hours.`

Comment (nataliekwong) [reply docs/cloud/managing-airbyte-cloud/manage-schema-changes.md]: Great, updated 👍

**Review by malikdiarra — APPROVED** [review]

File: docs/cloud/managing-airbyte-cloud/manage-schema-changes.md [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
-:::
+Based on your configured settings for "Detect and propagate schema changes", Airbyte can automatically sync those changes or ignore them: 
+* **Propagate all changes** automatically propagates stream changes (additions or deletions) or column changes (additions or deletions) detected in the source
+* **Propagate column changes only** automatically propagates column changes detected in the source
+* **Ignore** any schema change, in which case the schema you’ve set up will not change even if the source schema changes until you approve the changes manually
+* **Pause connection** disables the connection from syncing further once a change is detected
+
+When a new column is detected and propagated, values for that column will be filled in for the updated rows. If you are missing values for rows not updated, a backfill can be done by completing a full refresh.
+
+When a column is deleted, the values for that column will stop updating for the updated rows and be filled with Null values.
```
Comment (alex-gron) [inline docs/cloud/managing-airbyte-cloud/manage-schema-changes.md:15]: I think that we will actually now delete these columns immediately, based on the related convo in Normalization.

File: docs/cloud/managing-airbyte-cloud/manage-schema-changes.md [hunk 3]
```diff
@@ -29,11 +44,10 @@ To review non-breaking schema changes:
...
-
-Breaking changes can only occur in the **Cursor** or **Primary key** fields.
-    
-:::
+Breaking schema changes occur when:
+* The data type of a field from the source changes
+* An existing primary key is removed from the source
+* An existing cursor is removed from the source
 
 To review and fix breaking schema changes:
```
Comment (alex-gron) [inline docs/cloud/managing-airbyte-cloud/manage-schema-changes.md:52]: What is the fix is the data type changes?

File: docs/cloud/managing-airbyte-cloud/manage-schema-changes.md [hunk 4]
```diff
@@ -69,3 +83,6 @@ In addition to Airbyte Cloud’s automatic schema change detection, you can manu
  3. In the **Activate the streams you want to sync** table, click **Refresh source schema** to fetch the schema of your data source.
 
  2. If there are changes to the schema, you can review them in the **Refreshed source schema** dialog.
```
[the remark this answers is not in the document]
Comment (nataliekwong) [reply docs/cloud/managing-airbyte-cloud/manage-schema-changes.md:85]: Oh nice! Our markdown system fixes this automatically as "4" but I've also updated it so it's clearer.

File: docs/cloud/managing-airbyte-cloud/manage-schema-changes.md (hunk 2 above)
Comment (nataliekwong) [reply docs/cloud/managing-airbyte-cloud/manage-schema-changes.md:15]: @alex-gron Was thinking about this more today after our chat - I agree that's the behavior we decided on. Does that mean the proposed changes will come with Destinations V2 - which I believe that's coming at the end of Q3? On the one hand, we could publish these with the foresight those changes will be coming, or re-publish with the updates when V2 is officially released. I'm leaning towards the latter but let me know if you think it's better to just update now!

Comment (alex-gron) [reply docs/cloud/managing-airbyte-cloud/manage-schema-changes.md:15]: Sorry for the late reply! I'm good with that option :)

File: docs/cloud/managing-airbyte-cloud/manage-schema-changes.md (hunk 3 above)
Comment (alex-gron) [reply docs/cloud/managing-airbyte-cloud/manage-schema-changes.md:50]: Ah, I had a typo in my original question.

I meant to ask  - what should a user do when they encounter a data type change? Is the only option in that case to run a reset?

**Pull request merged**